Library/Homebrew/cmd/audit.rb

```diff
@@ -6,6 +6,7 @@
 require "official_taps"
 require "cmd/search"
 require "date"
+require "vendor/method_source/method_source"
```

Comment:
It's pretty neat but it should be possible to extract the specific functionality into a utility method rather than include e.g. JRuby support and all that.